### ISSUE TITLE
Enable remapping hotkeys

### ISSUE THREAD

**Issue by PhilippMDoerner** [PhilippMDoerner/NimStoryFont#88]
WCAG demands that hotkey navigation allows remapping such keys.

**Comment by PhilippMDoerner** [comment]
Architecture:
- [ ] HotkeyDirective no longer binds specific keys to an Element that trigger a callback, it binds "actions" to those buttons/link. It itself should create the keydown-subscription that checks if an action should be triggered
- [ ] HotkeyDirective should come with a config that listens to keydown events on this specific element. Only if it's passed the "global" option should it listen to global keydowns
- [ ] HotkeyService provides a function that takes that action (and the element to listen to) and turns it into a keydown listener on that element
- [ ] There is no global modifier (ALT key) anymore, the defaults just come with Alt + <key> defaults pre-configured

**Comment by PhilippMDoerner** [comment]
Done
